### PR TITLE
Let's Merge One In Already

### PR DESCRIPTION
**Pull request by jbadlato**

### CODE REVIEW

File: src/bot.ts [hunk 1]
```diff
@@ -2,13 +2,24 @@
...
+import ai from 'leftovers-again/src/ai';
+const AI = ai.AI;
+import decisions from 'leftovers-again/src/decisions';
+type Decision = decisions.Decision;
+const MOVE = decisions.MOVE;
+const SWITCH = decisions.SWITCH;
+import battle from 'leftovers-again/src/battle';
+type State = battle.State;
+import pokemon from 'leftovers-again/src/model/pokemon';
+type PokemonData = pokemon.PokemonData;
```
Comment (jbadlato) [inline src/bot.ts]: This is the only ugly part IMO

File: src/bot.ts [hunk 2]
```diff
@@ -2,13 +2,24 @@
...
 /**
  * Your code is pre-built with a very simple bot that chooses a team, then
  * picks randomly from valid moves on its turn.
  */
-class Deepindigo {
+export default class DeepIndigo extends AI {
```
Comment (dpgil) [inline src/bot.ts]: i don't gaf how we do it either way, but i will just say that the `AI` class has a `team` function that isn't being enforced here, so somewhere type safety is being lost

Comment (jbadlato) [reply src/bot.ts]: Nah dude I don't think so, because we're extending a class, not implementing an interface

Comment (dpgil) [reply src/bot.ts]: okie

File: src/bot.ts (hunk 1 above)
Comment (dpgil) [reply src/bot.ts]: we can fix by going back to the single .d.ts file i believe

**Pull request merged**